Apromore-Extras/APMLogModule/src/main/java/org/apromore/apmlog/stats/CaseAttributeValue.java

```diff
@@ -75,7 +88,7 @@ public long getCases() {
     }
 
     public String getFrequency() {
```

Comment:
Maybe this should be renamed to "getPercentFormatted" to make it clear that it's the same value as "getPercent".